flake8_spellcheck/technical.txt

```diff
@@ -4,15 +4,19 @@ autocomplete
 autofill
 autotest
 backend
+bitwise
+bmp
 buildability
 changelist
 changelists
 checkboxes
 chipset
 cli
+config
 deserialize
 dev
 dynamodb
+erroring
```

Comment:
I'm not convinced this is a technical term. I agree its a term i've heard used by some people before, but I think it may dependent on the repo owner to decide whether its accepted or not.

Reply:
[erroring](https://en.wiktionary.org/wiki/erroring) seems to only be a word in some smaller community-made dictionaries (it isn't in merriam-webster or dictionary.com for example), but i've heard error used as a verb a *ton* in programming: "it errored", "stop erroring", "it will error", etc.

i can definitely remove it if you think it isn't appropriate though

Reply:
Let's keep it in then. I'm already considering adding a deny list equivalent to allow users to omit words they don't want to allow in the future anyway

Reply:
i think a deny list would be a fantastic idea